clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -580,21 +582,32 @@ public static Set<TopicPartition> generateRandomTopicPartitions(int numTopic, in
     }
 
     /**
-     * Assert that a future raises an expected exception cause type. Return the exception cause
-     * if the assertion succeeds; otherwise raise AssertionError.
+     * Assert that a future raises an expected exception cause type.
+     * This method will wait for the future to complete or timeout(15000 milliseconds) after a default period.
```

Comment:
nit: This method will wait for the future to complete or timeout(15000 milliseconds).